search_solr_tools.gemspec

```diff
@@ -8,7 +8,8 @@ Gem::Specification.new do |spec|
   spec.name          = 'search_solr_tools'
   spec.version       = SearchSolrTools::VERSION
   spec.authors       = ['Chris Chalstrom', 'Michael Brandt', 'Jonathan Kovarik', 'Luis Lopez', 'Stuart Reed', 'Julia Collins', 'Scott Lewis']
-  spec.email         = ['cchalstr@nsidc.org', 'mbrandt@colorado.edu', 'kovarik@nsidc.org', 'luis.lopezespinosa@colorado.edu', 'stuart.reed@colorado.edu', 'jcollins@nsidc.org', 'scott.lewis@nsidc.org']
+  spec.email         = ['cchalstr@nsidc.org', 'mbrandt@colorado.edu', 'Jonathan.Kovarik@colorado.edu', 'luis.lopezespinosa@colorado.edu', 'stuart.reed@colorado
```

Comment:
A few of these email addresses (Chris, Michael, and Stuart) aren't really valid anymore.  Obviously we still want to credit them, but do we perhaps want to remove their emails?

Reply:
I'll remove the emails as you suggest. Thanks!